docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+
+Otherwise, create a folder for a new project. From the terminal window, create an empty folder called `cmakeQuickStart`, navigate into it, and open VS Code in that folder by entering the following commands:
+
+```cmd
+mkdir cmakeQuickStart
+cd cmakeQuickStart
+code .
+```
+
+The `code .` command opens VS Code in the current working folder, which becomes your "workspace".
+
+### Create a CMake hello world project
+
+The CMake Tools extension can create the files for a basic CMake project for you. Open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Quick Start** command:
+
+![Create CMake quickstart](images/cpp/cmake-quickstart-command-palette.png)
+
+Enter a project name. This will be written to `CMakeLists.txt` and a few initial source files.
+
+Next, select **Executable** as the project type to create a basic source file (`main.cpp`) that includes a basic `main()` function.
+
+![Choose project type](images/cpp/cmake-choose-type.png)
+
+**Note:** If you had wanted to create a basic source and header file, you would have selected **Library** instead. But for this tutorial, **Executable** will do. If you are prompted to configure IntelliSense for the folder, you can either ignore the prompt or select **Allow**.
+
+This creates a hello world CMake project containing `main.cpp`, `CMakeLists.txt` (which tells the CMake tools how to build your project), and a folder named **build** for your build files:
+
+![Project contents](images/cpp/cmake-project-contents.png)
+
+### Select a kit
+
+Before you can use the CMake tools extension to build a project, you need to configure it to know about the compilers on your system. Do that by scanning for what CMake calls 'kits'. A kit represents a toolchain, which is the compilers, linkers, and other tools used to build your project. To scan for kits:
+
+1. Open the Command Palette (`kb(workbench.action.showCommands)`) and run **CMake: Select a Kit**. The extension will automatically scan for kits on your computer and create a list of compilers found on your system.
+
+1. Select the compiler you want to use. For example, depending on the compilers you have installed, you might see something like:
+
+    ![Select the kit](images/cpp/cmake-selectkit.png)
+
+## Configure Hello World
+
+There are two things you must do to configure your CMake project: select a kit (which you just did) and select a variant.  
+
+The kit you selected previously is shown in the status bar. For example:
+
+![Selected kit in status bar](images/cpp/cmake-kit-statusbar.png)
+
+To change the kit, you can click on the kit in the status bar, or run the **CMake: Select a kit** again from the Command Palette. If you don't see the compiler you're looking for, you can edit the `cmake-tools-kits.json` file in your project and add it manually.
```

Comment:
Does `cmake-tools-kits.json` get created automatically?  I didn't think so.  People might now know what this file is.  They can run the "Edit user-local CMake Kits" command to add the kit to the global list or create a `cmake-tools-kits.json` in the .vscode folder if they want to create a project-specific kit.  I'm not sure how much detail you want to go into here though.

Reply:
I added the step to run the cmake:edit user-local cmake kits command.